app/assets/stylesheets/framework/ui.scss

```diff
@@ -0,0 +1,79 @@
+html,
+body {
+  font-size: 16px;
+}
+
+main {
+  font-family: "Open Sans", sans-serif;
+}
+
+h1 {
+  font-size: 3rem;
+  margin: $double-margin 0;
+
+  @media screen and (max-width: $breakpoint) {
+    font-size: 2rem;
+    margin: $default-margin 0;
+  }
+}
+
+section {
+  margin-bottom: $double-margin;
+}
+
+button,
+input[type=submit],
+input[type=button],
+input[type=file] {
+  font-family: inherit;
+  -webkit-appearance: none;
+}
+
+.bttn {
+  border: 0;
+	border-radius: $radius;
+	cursor: pointer;
+	display: inline-block;
+	font-weight: bold;
+	line-height: 22px;
+  padding: 8px 20px;
+	text-decoration: none;
+
+	&:disabled,
+	&:disabled:hover,
+	&.disabled,
+	&.disabled:hover {
+		background-color: #ccc;
+		color: white;
+		cursor: wait;
+	}
+
+	&:focus {
+		outline: none;
+	}
+}
+
+.bttn-big {
+  padding: 10px 25px;
+}
+
+.bttn-primary {
+	@extend .bttn;
```

Comment:
- Line should be indented with spaces, not tabs
- Prefer using placeholder selectors (e.g. %some-placeholder) with @extend